src/dos.rs

```diff
@@ -1,40 +1,77 @@
-use rusty_asm::rusty_asm;
-
-pub fn print(s: *const u8) {
-    unsafe {
-        rusty_asm! {
-            let ptr: in("{dx}") = s;
-            clobber("ah");
-            clobber("al");
-            asm("volatile", "intel") {
-                "mov ah, 09h
-                int 21h"
-            }
-        }
-    }
-}
-
-pub fn print_character(c: u8) {
-    unsafe {
-        rusty_asm! {
-            let byte: in("{dl}") = c;
-            clobber("ah");
-            clobber("al");
-            asm("volatile", "intel") {
-                "mov ah, 02h
-                int 21h"
-            }
-        }
-    }
-}
-
-pub fn exit() {
-    unsafe {
-        rusty_asm! {
-            asm("volatile", "intel") {
-                "mov ah, 4Ch
-                int 21h"
-            }
-        }
-    }
+use rusty_asm::rusty_asm;
+
+pub fn print(s: *const u8) {
+    unsafe {
+        rusty_asm! {
+            let ptr: in("{dx}") = s;
+            clobber("ah");
+            clobber("al");
+            asm("volatile", "intel") {
+                "mov ah, 09h
+                int 21h"
+            }
+        }
+    }
+}
+
+pub fn print_character(c: u8) {
+    unsafe {
+        rusty_asm! {
+            let byte: in("{dl}") = c;
+            clobber("ah");
+            clobber("al");

```

Comment:
Does it make sense to just say `clobber("ax")` here, or is there a readability reason not to ?

Reply:
I asked @Serentty the exact same question XD since he wrote that part, I didn't change!

Reply:
I wrote it as two registers because I was using it as two values and therefore it seemed like two registers in my head. There's no good reason for it to be that way as far as I know, though.